### PR TITLE
[kjobctl] Add list interactive command

### PR DESCRIPTION
**Pull request by IrvingMg**
#### What type of PR is this?

/kind feature

#### What this PR does / why we need it:

Add `list interactive` command to kjobctl.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: cmd/experimental/kjobctl/pkg/cmd/list/list_interactive.go [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+
+	opts := metav1.ListOptions{
+		LabelSelector: constants.ProfileLabel,
+		FieldSelector: o.FieldSelector,
+		Limit:         o.Limit,
+	}
+
+	if len(o.ProfileFilter) > 0 {
+		opts.LabelSelector = fmt.Sprintf("%s,%s=%s", opts.LabelSelector, constants.ProfileLabel, o.ProfileFilter)
+	}
```
Comment (trasc) [inline cmd/experimental/kjobctl/pkg/cmd/list/list_interactive.go:152]:
```suggestion
	opts := metav1.ListOptions{
		LabelSelector: constants.ProfileLabel,
		Limit:         o.Limit,
	}

	if len(o.ProfileFilter) > 0 {
		opts.LabelSelector = fmt.Sprintf("%s=%s", constants.ProfileLabel, o.ProfileFilter)
	} else {
		opts.LabelSelector = constants.ProfileLabel
	}
```

nit: It's easier to follow and it makes more sense in my opinion.

File: cmd/experimental/kjobctl/pkg/cmd/list/list_interactive.go [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+		IOStreams:  streams,
+		Clock:      clock,
+	}
+}
+
+func NewInteractiveCmd(clientGetter util.ClientGetter, streams genericiooptions.IOStreams, clock clock.Clock) *cobra.Command {
+	o := NewInteractiveOptions(streams, clock)
+
+	cmd := &cobra.Command{
+		Use:                   "interactive [--selector key1=value1] [--field-selector key1=value1] [--all-namespaces]",
```
[the remark this answers is not in the document]
Comment (mbobrovskyi) [reply cmd/experimental/kjobctl/pkg/cmd/list/list_interactive.go]: Done

**Review by trasc — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**